codec-http2/src/test/java/io/netty/handler/codec/http2/DataCompressionHttp2Test.java

```diff
@@ -141,7 +141,7 @@ public void justHeadersNoData() throws Exception {
 
         runInChannel(clientChannel, () -> {
             clientEncoder.writeHeaders(ctxClient(), 3, headers, 0, true, newPromiseClient());
-            clientHandler.flush(ctxClient());
+            clientHandler.flush(ctxClient(), );
```

Comment:
There should be callable after `ctxClient(), `.

Reply:
Like I said this is not complete yet... I stopped to show the idea first :)

Reply:
Ah, alright. :)